README.md

```diff
@@ -3,8 +3,8 @@
 ### Configure the author name and email address to be used with your commits. Note that Git strips some characters (for example trailing periods) from user.name
 
 ```
-git config --global user.name "Sam Smith"
-git config --global user.email sam@example.com
+git config --global user.name "David"
+git config --global user.email david@manektech.com
```

Comment:
is this required ?

Reply:
Yes added for new credentials as old one are lost!